meta-balena-common/recipes-core/images/balena-image-flasher.bb

```diff
@@ -29,6 +29,8 @@ IMAGE_INSTALL = " \
     packagegroup-resin-flasher \
     "
 
+IMAGE_INSTALL_append = "${@oe.utils.conditional('SIGN_API','','',' cryptsetup dosfstools e2fsprogs-mke2fs libtss2-tcti-device lvm2-udevrules tpm2-tools',d)}"
```

Comment:
This is not necessary, it will be pulled as a dependency of `resin-init-flasher`

Reply:
Removed.